model/infinispan/src/main/java/org/keycloak/infinispan/module/KeycloakModule.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2025 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Based on Stian's email, we can remove these headers from all the new files you're adding.

Reply:
I'm still waiting for a reply on my questions around that. Until that is answered, I'd rather continue to add them until we purge them all eventually.

Reply:
ops, already removed them.